languages.toml

```diff
@@ -324,6 +324,28 @@ comment-token = "//"
 language-servers = [ "mint" ]
 indent = { tab-width = 2, unit = "  " }
 
+[[language]]
+name = "janet-simple"
+scope = "source.janet"
+injection-regex = "janet"
+file-types = ["cgen", "janet", "jdn"]
+shebangs = ["janet"]
+roots = ["project.janet"]
+comment-token = "#"
+indent = { tab-width = 2, unit = "  " }
+formatter = { command = "janet-format" }
+
+[language.auto-pairs]
+'"' = '"'
+'(' = ')'
+'[' = ']'
+'{' = '}'
+"`" = "`"
+
+[[grammar]]
+name = "janet-simple"
```

Comment:
The name of the `[[grammar]]` section needs to match the name defined in `grammar.js` (replacing hyphens with underscores) but the language definition doesn't need to be called `janet-simple` too:

```toml
[[language]]
name = "janet"
# ..
grammar = "janet-simple"

[[grammar]] # this part is unchanged
name = "janet-simple"
# ..
```

then the queries need to move to `runtime/queries/janet/*.scm`

Reply:
Thanks, I think I understand how to change things.